src/Combobox.jsx

```diff
@@ -224,42 +224,41 @@ var ComboBox = React.createClass({
           onChange={this._inputTyping}
           onKeyDown={this._inputKeyDown}
         />
-        <PopupComponent
+        {open && <PopupComponent
```

Comment:
`NB`: Could we elaborate on why we added `open &&` and removed `shouldRenderList` below?